Jenkinsfile

```diff
@@ -59,7 +59,8 @@ def examples() {
     'ivy-secrets': { assertIvyIsNotRunningInDemoMode() },
     'ivy-valve': { assertValve() },
     'ivy-custom-errorpage': { assertCustomErrorPage() },
-    'ivy-scaling': { assertIvyIsNotRunningInDemoModeOnPort(80) }
+    'ivy-scaling': { assertIvyIsNotRunningInDemoModeOnPort(80) },
```

Comment:
what's the difference between ivy-scaling and ivy-scaling nginx? in this case we need to give ivy-scaling a more specific name.

Reply:
Changed ivy-scaling to ivy-scaling-haproxy